src/data/algolia.js

```diff
@@ -0,0 +1,23 @@
+import { getConfig } from '@edx/frontend-platform';
+import algoliasearch from 'algoliasearch';
+
+const METERS_IN_ONE_MILE = 1609;
+const PRECISION_RADIUS_MILES = 50;
+
+// initialize Algolia workers
+const initializeSearchClient = () => algoliasearch(
+  getConfig().ALGOLIA_APP_ID,
+  getConfig().ALGOLIA_SEARCH_API_KEY,
+);
+
+const getLocationRestrictionFilter = (userCountry) => (
+  `NOT blocked_in:"${userCountry}" AND (allowed_in:"null" OR allowed_in:"${userCountry}")`
+);
+
+const getPrecisionRadius = () => METERS_IN_ONE_MILE * PRECISION_RADIUS_MILES;
```

Comment:
How are we setting precision criteria? Have we any certain requirement or ask?

Reply:
Nope, i am just replicating what prospectus team use: https://github.com/edx/prospectus/blob/master/src/packages/home-page/components/Recommendations/components/GeoRecommendations.jsx#L45

But your concern is valid. 
@zainab-amir do we need any product decision on this?

Reply:
We don't need to set precision criteria. We want to keep the same functionality we had before with just a check for functional cookies consent.